src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -79,6 +84,10 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
   const hasFreeSpace =
     videoParticipants.length < POSTERPAGE_MAX_VIDEO_PARTICIPANTS;
 
+  if (!hasRoom || !hasParticipants || isRoomLoading) {
+    return <Loading />;
```

Comment:
Could this `!hasRoom || !hasParticipants` make it so it looks like endless loading? Could "not found" message be more appropriate?

Reply:
Not really possible, but we gain nothing from these checks either way. So I removed them